chore: Update contributing docs (#1419)
Amsterdam/design-system: CONTRIBUTING.md

- [Submitting Pull Requests](#repeat-submitting-pull-requests)
- [Credits](#pray-credits)

## :book: Code of Conduct

We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
Read [our Code of Conduct](https://github.com/Amsterdam/.github/blob/main/CODE_OF_CONDUCT.md) if you haven’t already.

<!-- ## :bulb: Asking Questions

See our [Support Guide](https://github.com/jessesquires/.github/blob/main/SUPPORT.md). In short, GitHub issues are not the appropriate place to debug your specific project, but should be reserved for filing bugs and feature requests.

TODO: add SUPPORT.md, https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project -->

## :inbox_tray: Opening an issue

Before creating an issue, check if you are using the latest version of the project. If you are not up-to-date, see if updating fixes your issue first.

### Reporting security issues

Review our [Security Policy](https://github.com/Amsterdam/.github/blob/main/SECURITY.md). **Do not** file a public issue for security vulnerabilities.

### Bug reports and other issues

> For all other issues, you can [create an issue on GitHub](https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue).

Author: Updated the link. I don't think the link to the GitHub docs is really necessary to be honest, it's pretty self-explanatory imo.

Follow these guidelines when doing so:

> - **Do not open a duplicate issue.** Search through existing issues to see if your issue has previously been reported. If your issue exists, comment with any additional information you have. You may simply note "I have this problem too", which helps prioritize the most common problems and requests.

Author: I agree. Shall I remove the last line?

> - **Do not open a duplicate issue.** Search through existing issues to see if your issue has previously been reported. If your issue exists, comment with any additional information you have. You may simply note "I have this problem too", which helps prioritize the most common problems and requests.

Reviewer: Yes please, I can post a suggestion.

- **Prefer using [reactions](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)**, not comments, if you simply want to "+1" an existing issue.

- **Fully complete the provided issue template.** The bug report template requests all the information we need to quickly and efficiently address your issue. Be clear, concise, and descriptive.

## :repeat: Submitting pull requests

> Before opening a pull request for non-trivial changes, it is usually best to first open an issue to discuss the changes, or discuss your intended approach for solving the problem in the comments for an existing issue.

Author: Hm, I don't really think so. The original phrasing breaks it up in three parts, your change makes it harder to read imo.

- **Coordinate bigger changes.** For large and non-trivial changes, open an issue to discuss a strategy with the maintainers. Otherwise, you risk doing a lot of work for nothing!

- **Prioritize understanding over cleverness.** Write code clearly and concisely. Remember that source code usually gets written once and read often. Ensure the code is clear to the reader. The purpose and logic should be obvious to a reasonably skilled developer, otherwise you should add a comment that explains it.

- **Follow existing coding style and conventions.** Keep your code consistent with the style, formatting, and conventions in the rest of the code base. When possible, these will be enforced with a linter. Consistency makes it easier to review and modify in the future.

- **[Follow our Definition of Done](https://github.com/Amsterdam/design-system/blob/main/documentation/definition-of-done.md)**.

- **[Resolve any merge conflicts](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github)** that occur.

- **Promptly address any CI failures**. If your pull request fails to build or pass tests, please push another commit to fix it.

> ### Local development

Author: The idea was that you need local development when submitting a PR, but I agree that it makes more sense to separate the sections. That also makes the document more readable probably.

> ### Local development

Reviewer: Yes, better.

Read [our Code of Conduct](https://github.com/Amsterdam/.github/blob/main/CODE_OF_CONDUCT.md) if you haven’t already.

## :inbox_tray: Opening an issue

Before creating an issue, check if you are using the latest version of the project.
If you are not up-to-date, see if updating fixes your issue first.

### Reporting a security issue

**Do not** file a public issue for security vulnerabilities.
This may result in the vulnerability details being made public, creating an unsafe situation.
You can submit your report via [the Zerocopter Coordinated Vulnerability Disclosure (CVD) form](https://app.zerocopter.com/en/cvd/fc5dad1c-27ee-4571-880b-438dc672d178).
For more information, review our [Security Policy](https://github.com/Amsterdam/.github/blob/main/SECURITY.md).

### Bug reports and other issues

For all other issues, you can [create an issue on GitHub](https://github.com/Amsterdam/design-system/issues).

Follow these guidelines when doing so:

- **Do not open a duplicate issue.**
  Search through existing issues to see if your issue has previously been reported.
  If your issue exists, comment with any additional information you have.

>   You may simply note "I have this problem too", which helps prioritize the most common problems and requests.

Reviewer: ```suggestion
  Reacting with a ‘thumbs up’ helps us prioritize the most common problems and requests.
```
